{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -67,32 +67,64 @@ addopts = [
     "--import-mode=importlib",  # allow using test files with same name
 ]
 
-[tool.isort]
-include_trailing_comma = true
-multi_line_output = 3
-profile = "black"
-skip_glob = ["docs/*"]
-
 [tool.black]
 line-length = 120
-target-version = ['py38']
-include = '\.pyi?$'
-exclude = '''
-(
-  /(
-      \.eggs
-    | \.git
-    | \.hg
-    | \.mypy_cache
-    | \.tox
-    | \.venv
-    | _build
-    | buck-out
-    | build
-    | dist
-  )/
-)
-'''
```

Comment:
Why remove these here, but add a similar list for `ruff`?

Reply:
I checked the defaults for both tools:

Black uses `.gitignore`, ruff hardcodes its default to a shorter list.

Therefore it was never necessary for black, but is for Ruff.